integration/management-api-tests/pom.xml

```diff
@@ -215,5 +215,10 @@
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-model</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.skyscreamer</groupId>
+            <artifactId>jsonassert</artifactId>
+            <version>1.5.1</version>
```

Comment:
Move the dependency version to the root pom. Use a constant to define the version

Reply:
Moved to the root pom